### DISCUSSION TITLE
I need emergency help

### DISCUSSION THREAD

**Discussion by mahla94h** [postfixadmin/postfixadmin#743] [General]
my professor forgot his official email password (related to university). how can he receive his password?
it is really important for me 
so i will be so appreciate if you can help me.
best regards

**Comment by DavidGoodwin** [comment]
either ... 
 * update the mailbox.password value in the database for the user's mailbox, using a known hash
 * login into postfixadmin as an domain admin / global admin, and change the password on the user's mailbox
 * if you've got the right version of postfixadmin, go through the reset-your-password hoops.
